src/main/java/me/ShermansWorld/AlathraExtras/yeet/YeetCommand.java

```diff
@@ -0,0 +1,29 @@
+package me.ShermansWorld.AlathraExtras.yeet;
+
+import me.ShermansWorld.AlathraExtras.AlathraExtras;
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.util.Vector;
+import org.jetbrains.annotations.NotNull;
+
+public class YeetCommand implements CommandExecutor {
+
+
+    public YeetCommand(final JavaPlugin plugin) {
+        plugin.getCommand("yeet").setExecutor(this);
+    }
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
+        if (args.length != 0) {
```

Comment:
```suggestion
        if (args.length == 1) {
```
This is accepting edge cases with more than one args otherwise.